<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePage</class>
 <widget class="QWidget" name="GamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="player1nameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2nameLabel">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1_0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0_0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2_0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
